src/tpm2_wrap.c

```diff
@@ -172,6 +172,249 @@ int wolfTPM2_Init(WOLFTPM2_DEV* dev, TPM2HalIoCb ioCb, void* userCtx)
     return rc;
 }
 
+#ifdef WOLFTPM2_ENABLE_CSHARP
+WOLFTPM2_DEV *wolfTPM2_SimpleInit(void)
```

Comment:
Let's enable support for these API's always (with wrapper). And let's call them `wolfTPM2_New` and `wolfTPM2_Free`.